app/javascript/chat/__tests__/reportAbuse.test.jsx

```diff
@@ -19,24 +19,26 @@ describe('<ReportAbuse />', () => {
   });
 
   it('should render the component', () => {
-    const { queryByText, getByTestId } = render(
```

Comment:
`getByTestId` was not necessary. We can leverage `getByLabelText` which also encourages a11y.